detaileddesing.md

```diff
@@ -4,6 +4,10 @@ title: Detailed Design
 permalink: /detailed-design/
 ---
 
+## MACROS
+* [DecodeDshot2bit.md]({{ '/decode-dshot-2bit/' | relative_url }})
```

Comment:
No need to attach ".md" to the link text.

Reply:
Fixed